test/system/guest_views_changelog_test.rb

```diff
@@ -21,7 +21,7 @@ class GuestViewsChangelogTest < ApplicationSystemTestCase
     assert_text "Created by author1"
     assert_text "Published 1 day ago"
     assert_link "View", href: "https://exercism.io/new-exercise"
-    assert_selector "img[src='https://exercism.io/image.jpg']"
+    assert_selector "img[src='https://exercism.io/image.jpg']", visible: false
```

Comment:
What is the need for this change?

Reply:
Since we have a non existent image (`exercism.io/image.png`), without an alt tag, it wouldn't be visible.

Reply:
Ah. Cool. Yes, `https://assets.exercism.io/social/general.png`

Reply:
But `visible: false` is also cool if you prefer it.